### ISSUE TITLE
NVDA does not announce 'Suggested users list' names in MS Word search suggestion edit box

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13494]
### Steps to reproduce:
1. Launch word and open a document uploaded in One drive.
2. Navigate to review tab and activate new comment button.
3. Navigate to comment edit field and type "@".
4. Observe that a group of contact list will appear and navigate with down arrow.
5. Observe the announcement of NVDA.

### Actual behavior:
When we type "@", we will get suggestions, NVDA is not announcing the contact's name when we navigate through it. It is announcing as "@" for every suggestion. While navigating through the suggestions, It is announcing the content in edit field we type.

### Expected behavior:
NVDA should announce the suggestions available information and contact names while navigating through it.

### System configuration
Application: Word 

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
NVDA Version 2021.3.3

#### Windows version:
Window 11, Version 21H2 (OS Build 22000.556)

#### Does the issue still occur after restarting your computer?
yes

**Comment by CyrilleB79** [comment]
Hello
Please change the title of this issue.
You probably meant "NVDA" instead of "Talkback".
